lesson-5-recap/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Strange buttons, Lesson 5</title>
+  <link rel="stylesheet" href="main.css">
+</head>
+<body>
+  <div class='wrapper'>
+    <button id="strange-button">I'm a strange button</button>
+  </div>
+
+  <script type="text/javascript">
+    const buttonCmp = document.getElementById('strange-button');
+    const wrapperCmp = document.getElementsByClassName('wrapper')[0];
+    const bodyCmp = document.getElementsByTagName('body')[0];
+    let buttonContent = buttonCmp.innerHTML;
+
+    const additionalCmp = document.createElement('button');
+    additionalCmp.classList.add('impossible');
+    additionalCmp.innerHTML = 'You can\'t press me!';
+    
+    // Tooltip
+    const tooltipCmp = document.createElement('span');
+    tooltipCmp.classList.add('tooltip');
+    tooltipCmp.innerHTML = 'Move out of the button!'
+    wrapperCmp.appendChild(tooltipCmp);
+
+    function toggleClasses() {
+      buttonCmp.classList.toggle('mouseover');
+      bodyCmp.classList.toggle('colored');
+      tooltipCmp.classList.toggle('visible');
+    }
+
+    function clickHandler() {
+      console.log('button was pressed');
+      buttonCmp.innerHTML = 'I\'m the pressed strange button';
```

Comment:
You can use such a construction : buttonContent = buttonCmp.innerHTML = `I'm the pressed strange button`;

Reply:
thank you for your comment! I prefer to use in JS only single quotes